omnigibson/utils/motion_planning_utils.py

```diff
@@ -133,7 +143,7 @@ def state_valid_fn(q):
 
     if solved:
         # try to shorten the path
-        ss.simplifySolution()
+        # ss.simplifySolution()
```

Comment:
Why not?

Reply:
Currently, if we simplify the solution, it potentially collides with an object. There may be two reasons.
1. OMPL's simplySolution does not guarantee collision-free simplification. It randomly samples points in trajectory and their intermediate points and removes those points. Perhaps @misoshiruseijin can explain more.
2. Interpolation in JointController and arm mass issue in Tiago.

Reply:
Aren't we re-interpolating to add waypoints now? @sujaygarlanka

Reply:
yes, removed this